yt/data_objects/static_output.py

```diff
@@ -131,7 +131,13 @@ class MutableAttribute:
 
     def __init__(self, display_array=False):
         self.data = weakref.WeakKeyDictionary()
-        self.display_array = display_array
+        # We can assume that ipywidgets will not be *added* to the system
+        # during the course of execution, and if it is, we will not wrap the
+        # array.
+        if display_array and find_spec("ipywidgets") is not None:
+            self.display_array = True
+        else:
+            self.display_array = False
```

Comment:
I have a small preference for the one-liner form, but it's not crucial (I've checked that it is black-compliant)
```suggestion
        self.display_array = display_array and find_spec("ipywidgets") is not None
```